### ISSUE TITLE
Content-Type header not set on Kafka event source events

### ISSUE THREAD

**Issue by lance** [knative/eventing-contrib#1206]
**Describe the bug**
When receiving an event from a Kafka event source, the `Content-Type` header is not set. The headers look like this.

```json
{
  "host": "event-display.kafka.svc.cluster.local",
  "user-agent": "Go-http-client/1.1",
  "content-length": "14",
  "accept-encoding": "gzip",
  "ce-id": "partition:1/offset:24",
  "ce-source": "/apis/v1/namespaces/kafka/kafkasources/kafka-source#knative-demo-topic",
  "ce-specversion": "1.0",
  "ce-subject": "partition:1#24",
  "ce-time": "2020-05-07T19:36:48.168Z",
  "ce-type": "dev.knative.kafka.event",
  "forwarded": "for=10.131.0.72;proto=http",
  "k-proxy-request": "activator",
  "kourier-random-header": "c439945e-9099-11ea-96ab-0a580a830047",
  "traceparent": "00-8d0a1c9e6e46f504b4746ead6c536e7e-7171749f04a8b8b5-00",
  "x-envoy-expected-rq-timeout-ms": "600000",
  "x-forwarded-for": "10.131.0.72, 10.131.0.79",
  "x-forwarded-proto": "http",
  "x-request-id": "1cc46236-12db-4836-874f-6fa16bf028e0"
}
```
Note that there is no `content-type`.

 **Expected behavior**
I expect messages sent over HTTP to have a `Content-Type` header.

 **To Reproduce**
Receive an event from a Kafka event source and observe the headers.

 **Knative release version**
Unknown

 **Additional context**
The events are sent here: https://github.com/knative/eventing-contrib/blob/master/kafka/source/pkg/adapter/message.go#L70

**Comment by slinkydeveloper** [comment]
Hi! I think this is not a bug, look at the discussion here: https://github.com/knative/eventing-contrib/issues/1050

Tl;dr what content type should i use, if when i receive a message from kafka, i don't have any?

**Comment by lance** [comment]
I think, as discussed in #1050, that this scenario should perhaps be addressed in the spec somehow.

**Comment by slinkydeveloper** [comment]
Try opening an issue on cloudevents/spec, but i feel this was already discussed and rejected because it's perfectly valid to send on all supported protocol bindings a payload without a content type, hence the spec don't want to be more restrictive
